json-rpc/integration-tests/src/lib.rs

```diff
@@ -1730,6 +1730,65 @@ impl PublicUsageTest for GetEventByVersionWithProofTest {
     }
 }
 
+pub struct GetResourcesTest;
+impl Test for GetResourcesTest {
+    fn name(&self) -> &'static str {
+        "jsonrpc::get-resource-test"
+    }
+}
+
+impl PublicUsageTest for GetResourcesTest {
+    fn run<'t>(&self, ctx: &mut PublicUsageContext<'t>) -> Result<()> {
+        let env = JsonRpcTestHelper::new(ctx.url().to_owned());
+        let address = format!("{:x}", diem_sdk::types::account_config::diem_root_address());
+        let response = env.send("get_resources", json!([address]));
+
+        let value = response.result.unwrap();
+        let resources = value.as_object().unwrap();
+        // check that the CurrencyInfo<XDX> resource looks ok as a sanity check
+        let currency_info = resources
+            .get("0x1::Diem::CurrencyInfo<0x1::XDX::XDX>")
```

Comment:
what it looks like when there are multiple types? because the type_params in StructTag is an array.

Reply:
Same syntax as generics in Rust; it has a comma separator. E.g.,

`0x1::Module::Type<0x2::M::TypeArg1, 0x3::N::TypeArg2>`